packages/open-lens/package.json

```diff
@@ -103,7 +103,7 @@
     "afterSign": "build/notarize.js",
     "extraResources": [
       {
-        "from": "templates/",
+        "from": "node_modules/@k8slens/resource-templates/templates/",
```

Comment:
Because the code in https://github.com/lensapp/lens/blob/master/packages/core/src/renderer/components/dock/create-resource/lens-templates.injectable.ts hasn't been changed

Reply:
Now that you said it: it might be already broken (since the monorepo restructure) .. need to verify before merging.